doc/README.md

```diff
@@ -13,4 +21,11 @@ npm i -g forgae
 
 ### Documentation
 
-[Documentation](developer-documentation/getting-started.md)
\ No newline at end of file
+[Documentation](developer-documentation/getting-started.md)
+
+### Development
```

Comment:
This can be changed to **Contribution** and should be placed somewhere different than the getting started.